Tecchly/DLOplanTool
# Dlo 30 add recommendations to latest feedback screen

### **Related Issue/Keyword:**
[DLO-30](https://trello.com/c/jf4aqulb/30-latest-feedback)
[DLO-67](https://trello.com/c/rQwwSNlw/67-leaving-recommendations) #41 
### **Description:**
change the style of commendations and add textual recommendations to the latest feedback screen
### **Testing:**

**Steps for manual testing:**

1. let somebody or yourself make some commends and comments on your project shared with them 
2. Go to the feedback page, wait a bit, and then your feedback should show correctly.

### **Checklist:**
